examples/MNIST/conda.yaml

```
@@ -14,4 +14,4 @@ dependencies:
   - pytorch_lightning>=1.0.2
   - pandas
   - gorilla
-  - mlflow
+  - mlflow @ git+https://github.com/mlflow/mlflow@master
```

This can be changed to version 1.14 now right?
```suggestion
  - mlflow>=1.14.0
```